### ISSUE TITLE
How do I  easily access the documentation/docstring of createVariable()?

### ISSUE THREAD

**Issue by jypeter** [CDAT/cdms#320]
I wanted to check some parameters of `cdms2.createVariable`, but there is no docstring...

`help(cdms2.createVariable)` just returns
```
Help on function createVariable in module cdms2.tvariable:

createVariable(*args, **kargs)
```
Trying `help(cdms2.tvariable.createVariable) `returns the same non-helpful result

Using search in the readthedoc was not very helpful and it took me some time to eventually find https://cdms.readthedocs.io/en/latest/manual/cdms_2.html?highlight=createvariable#variable-constructors

**Comment by dnadeau4** [comment]
That does not exist,  I will copy the __init__() documentation there.
https://github.com/CDAT/cdms/blob/master/Lib/tvariable.py#L173-L186

I might enhanced it a little too.

**Comment by jypeter** [comment]
`createVariable` is a very important function of cdms2, and (beginning) users should be able to easily find the available keywords and what they mean

**Comment by jypeter** [comment]
@jasonb5 I just had a similar problem with a _not exactly verbose_ `help(setAutoBounds)`. Same thing with `getAutoBounds`

> Help on function setAutoBounds in module cdms2.axis:
> 
> setAutoBounds(mode)
> (END)

I guess all the functions lost their documentation at some point, which is too bad. Knowing what I was looking for, I went to the CDAT web site, and then searched the cdms2 documentation, and eventually found https://cdms.readthedocs.io/en/latest/manual/cdms_2.html?highlight=setAutoBounds#id3

It would be really useful to **get the docstrings back**. A temporary workaround would be to have at least a generic link in all the docstrings to the [cdms2 documentation site](https://cdms.readthedocs.io/en/latest/index.html), or the appropriate documentation

**Comment by jasonb5** [comment]
I agree the documentation is definitely lacking in areas, i'll need to make a pass through and update them.
